api/api-doc/raft.json

```diff
@@ -62,6 +62,38 @@
                ]
             }
          ]
+      },
+      {
+         "path": "/raft/read_barrier",
+         "operations": [
+            {
+               "method": "POST",
+               "summary": "Triggers read barrier for the given Raft group to ensure the schema is on the latest agreed state",
```

Comment:
schema is specific for group 0. And even for group 0, there are other things living in it than just schema.
So the API is more generic than waiting for latest schema.

Perhaps you can still mention schema but make it clear that it's just an example. I.e. "for the given Raft group. For example, you can use it with group 0 to ensure that the schema ..."

Also, "latest" is ambiguous, because new commands can be committed concurrently while the barrier is in progress, and it might or might not see them. What does "latest" mean in that case?